Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -1852,8 +1865,8 @@ public void getShortUrl(String alias, String channel, String feature, String sta
      * @see BranchLinkData#putStage(String)
      * @see BranchLinkData#putParams(String)
      * @see BranchLinkCreateListener
-     *
-     * @deprecated use {@link BranchShortLinkBuilder} instead.
+     * @deprecated Use {@link BranchUniversalObject } to create links to your content.
+     * For more details on content sharing please visit <a href="https://gist.github.com/sojanpr/158485479a1bfc0aa7e5">BranchUniversalObject</a>
      */
     @Deprecated
```

Comment:
These deprecations are there from 1.8.6 onwards (Since the addition of `BranchShortLinkBuilders` for URL creation) .Only thing changed in this version is the deprecation message ( pointed to BOU instead of `BranchShortLinkBuilder`) . So just wondering whether we need  to remove these deprecations?